### ISSUE TITLE
Can't capture the last x seconds only

### ISSUE THREAD

**Issue by ovidiubarbu** [MathewSachin/Captura#543]
Hi Mathew,
I just got to test this implementation, and maybe I am not using it properly, but it does seem to record everything since I start it, not the seconds specified on command.

What I do:

1. Start a cmd and type: captura-cli start --encoder roll:10 ( as per [this](https://github.com/MathewSachin/Captura/commit/f2babe98707b80afd1416928e451841c30cd65ca))
2. Press q to quit
3. Open the saved mp4 file

Expected result: Last 10 seconds are recorded
Actual result: everything is recorded, since I have started the command

Would you tell me if I am doing anything wrong?

**Comment by MathewSachin** [comment]
There was a change in how rolling recording (now called replay recording) is used from the command-line.
See https://github.com/MathewSachin/Captura/blob/master/docs/Cmdline/Verb-Start.md

You should use something like `captura-cli start --encoder sharpavi:0 --replay:10`

**Comment by ovidiubarbu** [comment]
Thank you for looking into this Matt. I am still having trouble launching it, even with the latest beta. It seems the option is not recognized:

_**C:\Program Files (x86)\Captura>captura-cli start --encoder sharpavi:0 --replay:10
Captura 9.0.0.0
(c) 2018 Mathew Sachin
ERROR(S):
Option 'replay:10' is unknown.
USAGE:
Record 10 seconds with cursor and keystrokes and audio from first speaker output.:
captura-cli start --cursor --keys --length 10
Record specific region:
captura-cli start --source 100,100,300,400
Record as Avi to out.avi:
captura-cli start --encoder sharpavi:0 --file out.avi**_

**Comment by MathewSachin** [comment]
Oops! I made a typo. There shouldn't be a `:` between replay and 10. See the above comment again.

**Comment by ovidiubarbu** [comment]
That's right! It seems to be working now, though I see a few issues. I I can open a new issue report if you want.

1. 1st monitor clips on and off
2. after I stop the monitoring with q, the last file, which would be the most important, is corrupted. Do you have a spot where I can drop the files, or are these known issues?

Still testing on latest beta since on official release that option is not working

Workaround for broken file: "Instead of pressing q, press p then q". The screen still goes between black and content, only 1 screen of the 2 if recording both screens, If recording one screen at a time, will flip in and out all the time.
